### ISSUE TITLE
Runtime issue: Cannot read property 'data' of undefined

### ISSUE THREAD

**Issue by vavrapavel** [dequelabs/axe-core#2944]
Ran a full scan on instance of an internally hosted enterprise application Polarion (GUI done via GWT): https://polarion.plm.automation.siemens.com/products/polarion-alm

axe-core version: `4.2.0`
axe-extension version: `4.9.3`

```
TypeError: Cannot read property 'data' of undefined
    at Xl._l [as after] (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:289593)
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:324563
    at Array.forEach (<anonymous>)
    at ec.after (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:324396)
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:332384
    at Array.map (<anonymous>)
    at e.value (chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:332240)
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:341091
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:97118
    at chrome-extension://lhdoppojpmngadmnindnejefpokejbdd/vendor.bundle.js:56:100990
```

**Comment by straker** [comment]
Thanks for the issue. We'll look into it.

**Comment by srayos** [comment]
Good morning! I am an engineer with Dell Technologies and our teams are experiencing this error, too. Can you please provide us with a status update? Is there any work-around at this point? Thank you!

**Comment by gdkraus** [comment]
We are experiencing the same issue with some of our internally hosted apps, but not all of our apps.

**Comment by gdkraus** [comment]
If it helps in diagnosing this, for us it seems to happen with apps with iframes, but not all apps with iframes. If I load two of our internal apps which have the same structure for the content in the parent frame but the single iframe in the app is different, if the iframe has a lot of contents the error occurs. If the iframe doesn't have much content it doesn't happen. When I get a chance I'll try to debug it further for the root cause in our apps.

Note, all of this definitely started happening with one of the latest axe devtools releases as we have never seen this error before.

**Comment by gdkraus** [comment]
When the error occurs, in the console I get the following message.

[Violation] 'setTimeout' handler took 499ms

**Comment by straker** [comment]
I've passed the ticket along to the appropriate team. They are looking into it now, but I'm not sure how long it will take to find the issue and get a fix out.

**Comment by dylanb** [comment]
We need to be able to reproduce this issue so if you find it on a public page, please post that URL here

**Comment by dylanb** [comment]
ok, looks like we have found a culprit, test pages would still be useful

**Comment by gdkraus** [comment]
I'll keep looking for some. I'm happy to test on our internal apps if you need confirmation of a fix.

**Comment by gdkraus** [comment]
@dylanb if you have access to Office 365 as a web app, I get the same error when running it in Word.

**Comment by srayos** [comment]
> ok, looks like we have found a culprit, test pages would still be useful

https://dell.bravais.com/s/VLCk9QOuMNHMpCMKYcYd

**Comment by dylanb** [comment]
This issue should now be fixed with the 4.17 release, closing
